src/components/FeedInfo.tsx

```diff
@@ -212,8 +213,15 @@ export class FeedInfo extends React.Component<Props, FeedInfoState> {
             this.handleInviteCode(inviteCode);
             return;
         }
-        const feedUrl = link;
-        this.handleFeedUrl(feedUrl);
+        if (urlUtils.isFelfeleResource(link)) {
+            this.handleFeedUrl(link);
+            return;
+        }
+        const feedUrl = urlUtils.getCanonicalUrl(link);
```

Comment:
While this fixes the bug with the links without protocol, this is maybe not the best way to solve it:
1. The `if` is not really necessary, because `getCanonicalUrl` should never return `undefined` or `null`
2.  The root cause of this problem is in the `fetchFeedFromUrl` function from `feedHelpers.ts`, where we also get the canonical url, but then don't use it.

https://github.com/felfele/felfele/blob/5d8aed38849d293a76afa5703e0c2f5dfdac43f7/src/helpers/feedHelpers.ts#L50

I would fix there, because that function we use in other code as well.

Reply:
fixed in 8442e7e